pkg/phase.go

```diff
@@ -84,6 +87,18 @@ func (p *Phase) Exec(ctx context.Context, bp crv1alpha1.Blueprint, action string
 	return p.f.Exec(ctx, tp, p.args)
 }
 
+func renderFuncArgs(
+	funcName string,
+	args map[string]interface{},
+	tp param.TemplateParams) (map[string]interface{}, error) {
+	// let wait handle its own go template + jsonpath mixed arguments
+	if strings.ToLower(funcName) == "wait" {
```

Comment:
Can we use the declared name from function pkg - `function.WaitFuncName`

Reply:
No, it will lead to an import loop.